### PR TITLE
Add XR Interaction Toolkit and Oculus support

### PR DESCRIPTION
**Pull request by brycekahle**
- Updates TestXRScene by making the Sphere interactable. You can pull the bouncing sphere to you by aiming and pressing the grip button. Tested using Oculus Rift CV1 w/ Touch.
- Adds teleportation and snap turning to TestXRScene. Teleport by aiming where you want to go and pressing the grip button.

### CODE REVIEW

File: unity-project/ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -12,12 +12,12 @@ PlayerSettings:
   targetDevice: 2
   useOnDemandResources: 0
   accelerometerFrequency: 60
-  companyName: DefaultCompany
+  companyName: Nexus Aurora
   productName: unity-project
   defaultCursor: {fileID: 0}
   cursorHotspot: {x: 0, y: 0}
   m_SplashScreenBackgroundColor: {r: 0.13725491, g: 0.12156863, b: 0.1254902, a: 1}
-  m_ShowUnitySplashScreen: 0
```
Comment (DeadlyMagikarps) [inline unity-project/ProjectSettings/ProjectSettings.asset:20]: Change this. Otherwise it will always show that popup when launching Play

Comment (brycekahle) [reply unity-project/ProjectSettings/ProjectSettings.asset:20]: Strange...I didn't manually change anything.

Comment (brycekahle) [reply unity-project/ProjectSettings/ProjectSettings.asset:20]: I cannot unset this in the UI. It is greyed out for me. This [article](https://support.unity3d.com/hc/en-us/articles/212165803-Why-can-t-I-remove-the-Splash-Screen-from-my-build-) says that Personal users cannot turn it off.

Comment (DeadlyMagikarps) [reply unity-project/ProjectSettings/ProjectSettings.asset:20]: I just saw this is in a different folder. haha. Take a look at my PR #18 

The line number is there to change that.

**Comment by DeadlyMagikarps** [comment]
@RMSteenwijk Can we move this to a new feature branch instead for testing. I think we all need a discussion in which type we want to use. I don't think we can have both. There will be conflicts. 

In regards to this being all beta, are we expected to ship this demo as just a beta? I guarantee there will be bugs unless it's all fixed and stable by October 2020 if Unity updates and XR updates alongside OpenVR

**Comment by DeadlyMagikarps** [comment]
We can name it features/xr-interaction

**Comment by Sch1nken** [comment]
Semi-related: There are news regarding OpenXR (https://store.steampowered.com/newshub/app/250820/view/2396425843528787269)

**Comment by RMSteenwijk** [comment]
> We can name it features/xr-interaction

Seems like the best option for now. We having back and forth over this dicussion

File: unity-project/ProjectSettings/ProjectSettings.asset (hunk 1 above)
Comment (RMSteenwijk) [reply unity-project/ProjectSettings/ProjectSettings.asset:20]: Isn't this a paid option of unity to disable the splash screen.

Comment (brycekahle) [reply unity-project/ProjectSettings/ProjectSettings.asset:20]: @RMSteenwijk yes, it is.

Comment (DeadlyMagikarps) [reply unity-project/ProjectSettings/ProjectSettings.asset:20]: You know, the more I look at this, the more I think this isn't the steamvr splash screen for unity like I thought it was. Lol.

Not sure who has unity pro. I definitely don't have pro version...

**Pull request merged**